tests/sentry/plugins/bases/test_issue2.py

```diff
@@ -14,31 +14,31 @@
 from sentry.testutils.helpers.datetime import iso_format, before_now
 
 
-class TestPluginWithFields(IssueTrackingPlugin2):
+class PluginWithFields(IssueTrackingPlugin2):
```

Comment:
@joshuarli is this rename happening because of a name clash?

Reply:
No. Pytest's default test collection behavior is to look at everything with a Test prefix. It tries to collect these as tests, and they aren't:

```
/Users/joshua.li/dev/sentry/sentry/tests/sentry/plugins/bases/test_issue2.py:17: PytestCollectionWarning: cannot collect test class 'TestPluginWithFields' because it has a __new__ constructor (from: tests/sentry/plugins/bases/test_issue2.py)
  class TestPluginWithFields(IssueTrackingPlugin2):
```

Since there's only two occurences I opted to not change this default test collection behavior and just rename those things.